src/lang/res_zh-CN.rc

```diff
@@ -318,6 +329,10 @@ BEGIN
 
     IDS_COPYINGTITLE            "正在复制..."
 
+JAPANBEGIN
+    IDS_KK_COPY                 "コピー先:"
```

Comment:
I doubt whether these JAPANBEGIN's were necessary. It worked well without them and I suggest to remove all JAPANBEGIN and JAPANEND's all together.